pkg/xsd/types.go

```diff
@@ -287,9 +289,43 @@ var staticTypes = map[string]staticType{
 	"hexBinary":          "string",
 }
 
+var (
+	staticTypeImports = map[string]string{}
+	staticTypeUsed    = map[string]struct{}{}
+)
+
+func AddStaticTypeOverride(override string) {
+	parts := strings.SplitN(override, "=", 2)
+	typeParts := strings.SplitN(parts[1], ":", 2)
+
+	typeName := parts[0]
+
+	staticTypes[typeName] = staticType(typeParts[0])
+
+	if len(typeParts) == 2 {
+		staticTypeImports[typeName] = typeParts[1]
+	}
+}
+
+func GetStaticTypeImports() []string {
+	imports := []string{}
+
+	for name, mod := range staticTypeImports {
+		if _, found := staticTypeUsed[name]; found {
+			imports = append(imports, mod)
+		}
+	}
+
+	sort.Strings(imports)
+
+	return imports
+}
+
 func StaticType(name string) staticType {
 	typ, found := staticTypes[name]
 	if found {
+		staticTypeUsed[name] = struct{}{}
```

Comment:
I am afraid that using `staticTypeUsed` as a global variable would break with the following corner-case scenario:

Consider we have multiple cross-referencing schema files we convert at once. And not all of these files use the override, but all would produce golang code that would import it.